recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install --no-deps .
```

Comment:
The missing six was entrypoints-related. Since the entrypoints didn't get built properly, it was running using the build env instead of the test env, so test env packages were unavailable.